html/semantics/embedded-content/media-elements/interfaces/HTMLElement/HTMLMediaElement/removeTextTrack.html

```diff
@@ -0,0 +1,76 @@
+<!doctype html>
+<title>HTMLMediaElement.removeTextTrack</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<div id=log></div>
+<script>
+var video = document.createElement('video');
+test(function(){
+    var t = video.addTextTrack('captions');
+    assert_equals(video.textTracks, video.textTracks);
+    assert_equals(video.textTracks.length, 1);
+    video.removeTextTrack(t);
+    assert_equals(video.textTracks, video.textTracks);
+    assert_equals(video.textTracks.length, 0);
+}, document.title + ' captions');
```

Comment:
I think it's probably fine to only test one of these cases, but if you do want to keep them all, you can do something like `['captions', 'subtitles', ...].forEach(function(kind) { test(...) })` to avoid repeating the body of the test so many times.